### PR TITLE
Helm-388: Rename Helm in the Helm docs

### PR DESCRIPTION
**Pull request by Bonrob2**
# Pull Request Check Sheet

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new feature, is there documentation?
* [ ] If this is a new feature or substantial change, are there tests that cover it?

# Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

Once the reviewer(s) accept the PR and the branch passes continuous integration in Bamboo, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

# External References

* JIRA (Issue Tracker): HELM-388
* Continuous Integration: [CircleCI](https://circleci.com/gh/OpenNMS/opennms-helm)

### CODE REVIEW

File: docs/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -24,7 +24,7 @@
 * xref:datasources:entity_datasource.adoc[]
 * xref:datasources:flow_datasource.adoc[]
 
-.Contribute to {page-component-title}
+.Contribute to the {page-component-title}
```
Comment (emarsh1995) [inline docs/modules/ROOT/nav.adoc:27]: Perhaps we could shorten this to "Contribute to the Plugin"?

File: docs/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 :data-uri:
 :icons:
 
 [.lead]
-Welcome to the {page-component-title} {page-version} documentation.
-Here, you will find information and guides to help you learn about {page-component-title} and start exploring its features.
-Learn how to set up and manage your {page-component-title} environment, from installation to creating your first dashboard and using advanced filtering.
+Welcome to the {page-component-title} documentation.
+Here, you will find information and guides to help you learn about the plugin and start exploring its features.
+Learn how to set up and manage your plugin environment, from installation to creating your first dashboard and using advanced filtering.
```
Comment (emarsh1995) [inline docs/modules/ROOT/pages/index.adoc:13]:
```suggestion
Learn how to set up and manage your environment, from installing the plugin to creating your first dashboard and using advanced filtering.
```

File: docs/modules/about/pages/introduction.adoc [hunk 3]
```diff
@@ -1,34 +1,34 @@
 
 :imagesdir: ../assets/images
 
-= About {page-component-title}
+= About the {page-component-title}
 
-{page-component-title} is an application that lets you create flexible dashboards to visualize and interact with data stored by OpenNMS.
+The {page-component-title} is an application that lets you create flexible dashboards to visualize and interact with data stored by OpenNMS.
```
Comment (emarsh1995) [inline docs/modules/about/pages/introduction.adoc:6]:
```suggestion
The {page-component-title} lets you create flexible dashboards to visualize and interact with data stored by OpenNMS.
```

File: docs/modules/about/pages/introduction.adoc [hunk 4]
```diff
@@ -1,34 +1,34 @@
...
 It works with https://grafana.com[Grafana] to customize and display <<fault, fault management>> and <<performance, performance management>> data from OpenNMS https://www.opennms.org[Horizon] and https://www.opennms.com[Meridian].
 
 You can create and customize dashboards to display the data that you want to see--alarms, outages, key performance indicators--in a way that best meets your needs.
 
-.Sample {page-component-title} dashboard
-image::helm-sample-dash.png["{page-component-title} displaying a sample dashboard, populated with bar graphs and pie charts."]
+.Sample plugin dashboard
+image::helm-sample-dash.png["The {page-component-title} displaying a sample dashboard, populated with bar graphs and pie charts."]
```
Comment (emarsh1995) [inline docs/modules/about/pages/introduction.adoc:12]:
```suggestion
image::helm-sample-dash.png["Grafana displaying a sample dashboard, populated with bar graphs and pie charts."]
```

File: docs/modules/datasources/pages/entity_datasource.adoc [hunk 5]
```diff
@@ -62,7 +62,7 @@ To find a list of valid attributes, see your OpenNMS server's `\http://HOSTNAME:
 
 Follow these steps to filter nodes by a custom asset record:
 
-. On your {page-component-title} dashboard, create a variable named `region`, and set its datasource to an entity datasource that points to your server.
+. On your plugin dashboard, create a variable named `region`, and set its datasource to an entity datasource that points to your server.
```
Comment (emarsh1995) [inline docs/modules/datasources/pages/entity_datasource.adoc:65]:
```suggestion
. On your Grafana dashboard, create a variable named `region`, and set its datasource to an entity datasource that points to your server.
```

File: docs/modules/getting_started/pages/basic_walkthrough.adoc [hunk 6]
```diff
@@ -3,31 +3,31 @@
 
 = Create a Dashboard
 
-This section demonstrates how to create a simple dashboard in {page-component-title}.
+This section demonstrates how to create a simple dashboard in the {page-component-title}.
```
Comment (emarsh1995) [inline docs/modules/getting_started/pages/basic_walkthrough.adoc:6]:
```suggestion
This section demonstrates how to create a simple dashboard using the {page-component-title}.
```

File: docs/modules/getting_started/pages/basic_walkthrough.adoc [hunk 7]
```diff
@@ -3,31 +3,31 @@
...
 
 You must have the following pre-requisites to create a dashboard:
```
Comment (emarsh1995) [inline docs/modules/getting_started/pages/basic_walkthrough.adoc:8]:
```suggestion
You must have the following set up to create a dashboard:
```

File: docs/modules/reference/pages/how-to-build-with-docker.adoc [hunk 8]
```diff
@@ -17,15 +17,15 @@ docker build -t myopennmshelmimage .
 [source, console]
 docker run -p 3000:3000 -t myopennmshelmimage
 
-. Log in to your {page-component-title} instance with the default admin credentials (`admin`/`admin`).
-. Navigate to *Plugins* and enable the *OpenNMS Helm* plugin.
+. Log in to your Grafana plugin instance with the default admin credentials (`admin`/`admin`).
+. Navigate to *Plugins* and enable the *OpenNMS plugin for Grafana*.
```
Comment (emarsh1995) [inline docs/modules/reference/pages/how-to-build-with-docker.adoc:21]:
```suggestion
. Navigate to *Plugins* and enable the *OpenNMS Plugin for Grafana*.
```

File: docs/modules/reference/pages/how-to-build-with-docker.adoc [hunk 9]
```diff
@@ -17,15 +17,15 @@ docker build -t myopennmshelmimage .
 [source, console]
 docker run -p 3000:3000 -t myopennmshelmimage
 
-. Log in to your {page-component-title} instance with the default admin credentials (`admin`/`admin`).
-. Navigate to *Plugins* and enable the *OpenNMS Helm* plugin.
+. Log in to your Grafana plugin instance with the default admin credentials (`admin`/`admin`).
```
Comment (emarsh1995) [inline docs/modules/reference/pages/how-to-build-with-docker.adoc:20]:
```suggestion
. Log in to your Grafana instance with the default admin credentials (`admin`/`admin`).
```

**Review by emarsh1995 — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/nav.adoc (hunk 1 above)
Comment (Bonrob2) [inline docs/modules/ROOT/nav.adoc:27]:
```suggestion
.Contribute to the plugin
```

**Comment by Bonrob2** [comment]
@synqotik , note there are still some instances of Helm in this directory, but I"m not sure if I should change them or if they are on the dev side. Please advise.

File: docs/modules/about/pages/introduction.adoc
Comment (emarsh1995) [inline docs/modules/about/pages/introduction.adoc:1]: At line 48, there's an instance of {page-component-title} without a preceding "the" that I can't reach to comment on.

File: docs/modules/getting_started/pages/basic_walkthrough.adoc [hunk 10]
```diff
@@ -3,31 +3,31 @@
...
 
 . Navigate to the home page of your Grafana instance.
 . In the left navigation menu, select menu:Configuration[Plugins]:
 +
 image::gf-plugins.png["Grafana menu displaying navigation options. The Plugins option is highlighted.", 250]
 
-. Select the *OpenNMS Helm* plugin.
+. Select the *OpenNMS Plugin for Grafana*.
 ** If the plugin is not listed, try restarting your Grafana server.
 ** If the issue persists, make sure the plugin is in the Grafana plugins directory (default: `var/lib/grafana/plugins`).
```
Comment (emarsh1995) [inline docs/modules/getting_started/pages/basic_walkthrough.adoc:27]:
```suggestion
** If the issue persists, make sure the plugin's files are in the Grafana plugins directory (default: `var/lib/grafana/plugins`).
```

File: docs/modules/getting_started/pages/basic_walkthrough.adoc [hunk 11]
```diff
@@ -69,7 +69,7 @@ The type of data that you can display in a dashboard depends on the datasource t
 
 Before you create a dashboard, you must set up the datasource (either xref:datasources:performance_datasource.adoc[performance], xref:datasources:flow_datasource.adoc[flow], or xref:datasources:entity_datasource.adoc[entity]).
```
Comment (emarsh1995) [inline docs/modules/getting_started/pages/basic_walkthrough.adoc:70]:
```suggestion
Before you create a dashboard, you must set up the datasource (xref:datasources:performance_datasource.adoc[performance], xref:datasources:flow_datasource.adoc[flow], or xref:datasources:entity_datasource.adoc[entity]).
```

File: docs/modules/getting_started/pages/basic_walkthrough.adoc [hunk 12]
```diff
@@ -69,7 +69,7 @@ The type of data that you can display in a dashboard depends on the datasource t
...
 For this walk-through, we will configure the entity datasource.
```
Comment (emarsh1995) [inline docs/modules/getting_started/pages/basic_walkthrough.adoc:71]:
```suggestion
For this walk-through, we will configure an entity datasource.
```

File: docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc [hunk 13]
```diff
@@ -1,7 +1,7 @@
 
 = Migrate From OpenNMS Data Source Plugin
```
Comment (emarsh1995) [inline docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc:2]:
```suggestion
= Migrate from OpenNMS Data Source Plugin
```

File: docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc [hunk 14]
```diff
@@ -11,35 +11,35 @@ To complete this migration, you must have the following:
...
-After you have installed {page-component-title}, you must enable the plugin in Grafana to make the provided panels and datasources available:
+After you have installed the {page-component-title}, you must enable it in Grafana to make the provided panels and datasources available:
 
 . Navigate to the home page of your Grafana instance.
 . In the left navigation menu, click menu:Configuration[Plugins]:
 +
 image::gf-plugins.png["Grafana menu displaying navigation options. The Plugins option is highlighted.", 250]
 
-. Select the *OpenNMS Helm* plugin.
+. Select the *OpenNMS plugin for Grafana*.
```
Comment (emarsh1995) [inline docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc:29]:
```suggestion
. Select *OpenNMS Plugin for Grafana*.
```

File: docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc [hunk 15]
```diff
@@ -11,35 +11,35 @@ To complete this migration, you must have the following:
...
 ** If the plugin is not listed, try restarting your Grafana instance.
 ** If the issue persists, make sure the plugin is in the Grafana plugins directory (default: `var/lib/grafana/plugins`).
```
Comment (emarsh1995) [inline docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc:31]:
```suggestion
** If the issue persists, make sure the plugin files are in the Grafana plugins directory (default: `var/lib/grafana/plugins`).
```

File: docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc [hunk 16]
```diff
@@ -11,35 +11,35 @@ To complete this migration, you must have the following:
...
 You can also choose *Update* to download the latest version of the plugin.
 
 == Migrate datasources
 
-After you enable the {page-component-title} plugin, you can convert your existing datasources to use the OpenNMS Performance type.
+After you enable the plugin, you can convert your existing datasources to use the OpenNMS Performance type.
 Note that when switching the type, you may need to re-enter the datasource's URL and authentication details.
 
-{page-component-title} provides two datasource types; the OpenNMS Performance type is equivalent to the previous "OpenNMS" datasource.
+The plugin provides two datasource types; the OpenNMS Performance type is equivalent to the previous "OpenNMS" datasource.
```
Comment (emarsh1995) [inline docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc:42]: I think we could move this line up to be the start of the previous paragraph. 🤔

File: docs/modules/reference/pages/how-to-build-with-docker.adoc [hunk 17]
```diff
@@ -1,47 +1,47 @@
...
+. Log in to your Grafana instance with the default admin credentials (`admin`/`admin`).
+. Navigate to *Plugins* and enable the *OpenNMS Plugin for Grafana*.
 
 IMPORTANT: The Grafana plugin directory location is changed from `/var/lib/grafana/plugins` to `/opt/grafana/plugins`.
 This is because the base Grafana image defines `/var/lib/grafana` as a volume, which cannot be changed.
 
 == Advanced build
 
-If you want to build a {page-component-title} image based on a specific GitHub fork or branch, you can use `--build-arg`:
+If you want to build a plugin image based on a specific GitHub fork or branch, you can use `--build-arg`:
```
Comment (emarsh1995) [inline docs/modules/reference/pages/how-to-build-with-docker.adoc:28]: I think in this case, using the full name makes more sense to reduce ambiguity (i.e. to say we're not referring to _a_ plugin, but _our_ plugin).

```suggestion
If you want to build {page-component-title} based on a specific GitHub fork or branch, you can use `--build-arg`:
```

File: docs/modules/reference/pages/how-to-build-with-docker.adoc [hunk 18]
```diff
@@ -51,5 +51,5 @@ The URL for your Horizon or Meridian datasource is `\http://opennms:8980/opennms
 
 Horizon and Meridian include templates from which you can create PDF reports.
 They use an existing Grafana dashboard (see https://docs.opennms.com/horizon/31/operation/deep-dive/database-reports/templates/grafana.html[Grafana Dashboard] in the main OpenNMS documentation).
-If you want to use this feature, and you are running {page-component-title} through Docker, you must complete additional configuration.
+If you want to use this feature, and you are running the plugin through Docker, you must complete additional configuration.
```
Comment (emarsh1995) [inline docs/modules/reference/pages/how-to-build-with-docker.adoc:54]:
```suggestion
If you want to use this feature, and you are running the {page-component-title} through Docker, you must complete additional configuration.
```

File: src/dashboards/about.json [hunk 19]
```diff
@@ -32,7 +32,7 @@
...
+          "content": "# About the OpenNMS Plugin for Grafana\n\nthe plugin is an application that allows users to create flexible dashboards to interact with data stored by OpenNMS. The plugin works with Grafana to display and customize data from OpenNMS. Create and customize dashboards to display the data you want to see - alarms, outages, key performance indicators - in a way that best  meets the needs of your workflow and staff.\n\n## Datasources\n\n * Performance - retrieve performance metrics (i.e., data collection and/or response time metrics) from the Measurements API in OpenNMS.\n * Flow - retrieve application- and conversation-related metrics derived from flow data stored by OpenNMS.\n * Entity - retrieve entity model objects from OpenNMS. It currently supports alarms and nodes.\n\n## Dashboard Template Repositories\n\nYou can use the links below to find some sample dashboard templates that can be imported into your Grafana instance.\n\n * https://grafana.com/grafana/dashboards?search=opennms\n * https://github.com/opennms-forge/grafana-dashboards\n ",
```
Comment (emarsh1995) [inline src/dashboards/about.json:35]:
```suggestion
          "content": "# About the OpenNMS Plugin for Grafana\n\nThe plugin is an application that allows users to create flexible dashboards to interact with data stored by OpenNMS. It works with Grafana to display and customize data from OpenNMS. You can create and customize dashboards to display the data you want to see - alarms, outages, key performance indicators - in a way that best meets the needs of your workflow and staff.\n\n## Datasources\n\n * Performance: Retrieve performance metrics (data collection and response time metrics) from the OpenNMS Measurements API.\n * Flow: Retrieve application- and conversation-related metrics derived from flow data stored by OpenNMS.\n * Entity: Retrieve entity model objects from OpenNMS. It currently supports alarms and nodes.\n\n## Dashboard Template Repositories\n\nYou can use these links to find some sample dashboard templates that can be imported into your Grafana instance:\n\n * https://grafana.com/grafana/dashboards?search=opennms\n * https://github.com/opennms-forge/grafana-dashboards\n ",
```

**Review by emarsh1995 — CHANGES_REQUESTED** [review]

File: docs/modules/installation/pages/plugin.adoc [hunk 20]
```diff
@@ -8,24 +8,25 @@ If you do not have an existing Grafana instance, see xref:package.adoc[].
 
 == Install via web UI
 
-Follow these steps to install the {page-component-title} plugin using the Grafana UI:
+Follow these steps to install the plugin using the Grafana UI:
 
-. Navigate to your Grafana dashboard.
-. Click the *Configuration* symbol and choose *Plugins*.
+
```
Comment (emarsh1995) [inline docs/modules/installation/pages/plugin.adoc:13]:
```suggestion
```

File: docs/modules/installation/pages/plugin.adoc [hunk 21]
```diff
@@ -8,24 +8,25 @@ If you do not have an existing Grafana instance, see xref:package.adoc[].
 
 == Install via web UI
```
Comment (emarsh1995) [inline docs/modules/installation/pages/plugin.adoc:9]:
```suggestion
== Install via Grafana UI
```

File: docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc [hunk 22]
```diff
@@ -11,35 +11,35 @@ To complete this migration, you must have the following:
 * A configured OpenNMS datasource.
 * One or more dashboards using the OpenNMS datasource.
 
-== Install {page-component-title}
+== Install the plugin
```
Comment (emarsh1995) [inline docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc:14]: This is me being pedantic, since we're talking about two plugins. :)

```suggestion
== Install the {page-component-title}
```

File: docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc [hunk 23]
```diff
@@ -11,35 +11,35 @@ To complete this migration, you must have the following:
...
 +
-NOTE: The Disable option indicates that the OpenNMS Helm plugin is already enabled.
+NOTE: The Disable option indicates that the OpenNMS {page-component-title} is already enabled.
 You can also choose *Update* to download the latest version of the plugin.
 
 == Migrate datasources
 
-After you enable the {page-component-title} plugin, you can convert your existing datasources to use the OpenNMS Performance type.
-Note that when switching the type, you may need to re-enter the datasource's URL and authentication details.
+The plugin provides two datasource types; the OpenNMS Performance type is equivalent to the previous "OpenNMS" datasource.
```
Comment (emarsh1995) [inline docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc:40]:
```suggestion
```

File: docs/modules/installation/pages/plugin.adoc [hunk 24]
```diff
@@ -8,24 +8,25 @@ If you do not have an existing Grafana instance, see xref:package.adoc[].
...
-. Search for "OpenNMS" and select the *OpenNMS Helm* plugin.
+. Search for "OpenNMS" and select *OpenNMS Plugin for Grafana*.
 . Click *Install*.
 
 == Install via CLI
 
-You can the `grafana-cli` tool to install {page-component-title}:
+You can the `grafana-cli` tool to install the {page-component-title}:
 
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/installation/pages/plugin.adoc:24]: I discovered yesterday that "console" isn't recognized as a syntax highlighter, so I'm updating these to "shell" as I come across them.

```suggestion
[source, shell]
```

File: docs/modules/installation/pages/source.adoc [hunk 25]
```diff
@@ -1,29 +1,29 @@
...
-== Build {page-component-title}
+== Build the plugin
 
-Follow these steps to build {page-component-title}:
+Follow these steps to build the plugin:
 
-. Download the source tree into a subfolder called `opennms-helm-app` in Grafana's plugin directory:
+. Download the source tree into a subfolder called `opennms-grafana-plugin-app` in Grafana's plugin directory:
 +
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/installation/pages/source.adoc:15]:
```suggestion
[source, shell]
```

File: docs/modules/installation/pages/source.adoc [hunk 26]
```diff
@@ -1,29 +1,29 @@
...
 mkdir -p /var/lib/grafana/plugins
 cd /var/lib/grafana/plugins
-git clone https://github.com/OpenNMS/opennms-helm.git opennms-helm-app
+git clone https://github.com/OpenNMS/grafana-plugin.git opennms-grafana-plugin-app
 ----
 
-. Compile {page-component-title}:
+. Compile the plugin:
 +
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/installation/pages/source.adoc:24]:
```suggestion
[source, shell]
```

File: docs/modules/installation/pages/source.adoc [hunk 27]
```diff
@@ -59,15 +59,15 @@ tar --exclude='.git' \
...
     --build-arg GRAFANA_VERSION=latest \
-    --build-arg OPENNMS_HELM_VERSION=SNAPSHOT \
-    -t myhelm:snapshot .
+    --build-arg OPENNMS_GRAFANA-PLUGIN_VERSION=SNAPSHOT \
+    -t mygrafana-plugin:snapshot .
 ----
 
 . Run Docker:
 +
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/installation/pages/source.adoc:68]:
```suggestion
[source, shell]
```

File: docs/modules/installation/pages/upgrading.adoc [hunk 28]
```diff
@@ -41,7 +41,7 @@ wizzy import datasources
 . Convert any `fault` datasources to `entity`:
 +
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/installation/pages/upgrading.adoc:43]:
```suggestion
[source, shell]
```

File: docs/modules/reference/pages/how-to-build-with-docker.adoc [hunk 29]
```diff
@@ -1,47 +1,47 @@
...
+This section describes how to build the {page-component-title} with Docker.
 
 == Simple build
 
-You can compile {page-component-title} from source with a Docker build image:
+You can compile the plugin from source with a Docker build image:
 
 . Check out the repository, compile the code, and install the plugin:
 +
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/reference/pages/how-to-build-with-docker.adoc:12]:
```suggestion
[source, shell]
```

File: docs/modules/reference/pages/how-to-build-with-docker.adoc [hunk 30]
```diff
@@ -1,47 +1,47 @@
...
-docker build -t myopennmshelmimage .
+docker build -t mygrafanapluginimage .
 
 . Run the image:
 +
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/reference/pages/how-to-build-with-docker.adoc:17]:
```suggestion
[source, shell]
```

File: docs/modules/reference/pages/how-to-build-with-docker.adoc [hunk 31]
```diff
@@ -51,5 +51,5 @@ The URL for your Horizon or Meridian datasource is `\http://opennms:8980/opennms
 
 Horizon and Meridian include templates from which you can create PDF reports.
 They use an existing Grafana dashboard (see https://docs.opennms.com/horizon/31/operation/deep-dive/database-reports/templates/grafana.html[Grafana Dashboard] in the main OpenNMS documentation).
-If you want to use this feature, and you are running {page-component-title} through Docker, you must complete additional configuration.
+If you want to use this feature, and you are running the {page-component-title} through Docker, you must complete additional configuration.
 Refer to Grafana's https://grafana.com/docs/grafana/latest/administration/image_rendering/#remote-rendering-service[Remote rendering service documentation] for more information.
```
Comment (emarsh1995) [inline docs/modules/reference/pages/how-to-build-with-docker.adoc:55]:
```suggestion
Refer to Grafana's https://grafana.com/docs/grafana/latest/administration/image_rendering/#remote-rendering-service[remote rendering service documentation] for more information.
```

File: docs/modules/contributing/pages/developing.adoc [hunk 32]
```diff
@@ -22,33 +22,33 @@ Follow these steps to prepare your development environment:
 app_mode = development
 ...
 [plugins]
-allow_loading_unsigned_plugins = opennms-helm-app
+allow_loading_unsigned_plugins = opennms-grafana-plugin-app
 ----
 
 . Use Git to check out the repository:
 +
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/contributing/pages/developing.adoc:30]:
```suggestion
[source, shell]
```

File: docs/modules/contributing/pages/developing.adoc [hunk 33]
```diff
@@ -22,33 +22,33 @@ Follow these steps to prepare your development environment:
...
-git clone https://github.com/OpenNMS/opennms-helm.git
+git clone https://github.com/OpenNMS/grafana-plugin.git
 
-. Navigate to your {page-component-title} directory:
+. Navigate to your plugin directory:
 +
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/contributing/pages/developing.adoc:35]:
```suggestion
[source, shell]
```

File: docs/modules/contributing/pages/developing.adoc [hunk 34]
```diff
@@ -22,33 +22,33 @@ Follow these steps to prepare your development environment:
...
-cd opennnms-helm
+cd opennnms-grafana-plugin
 
 . Install the build and runtime dependencies:
 +
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/contributing/pages/developing.adoc:40]:
```suggestion
[source, shell]
```

File: docs/modules/contributing/pages/developing.adoc [hunk 35]
```diff
@@ -22,33 +22,33 @@ Follow these steps to prepare your development environment:
...
 npm ci
 
-. Build {page-component-title} to prime your `dist` directory:
+. Build the plugin to prime your `dist` directory:
 +
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/contributing/pages/developing.adoc:45]:
```suggestion
[source, shell]
```

File: docs/modules/installation/pages/package.adoc [hunk 36]
```diff
@@ -25,19 +25,19 @@ deb-src https://debian.opennms.org stable main
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/installation/pages/package.adoc:25]:
```suggestion
[source, shell]
```

File: docs/modules/installation/pages/package.adoc [hunk 37]
```diff
@@ -50,7 +50,7 @@ sudo rpm --import https://yum.opennms.org/OPENNMS-GPG-KEY
 . Install the packages:
 +
 [source, console]
```
Comment (emarsh1995) [inline docs/modules/installation/pages/package.adoc:52]:
```suggestion
[source, shell]
```

**Review by emarsh1995 — CHANGES_REQUESTED** [review]
Some pedantic updates and changing instances of "console" to "shell" for syntax highlighting in code blocks.

**Comment by Bonrob2** [comment]
@synqotik , could you review to verify the name changes are correct? Thank you!

**Review by synqotik — APPROVED** [review]
Looks good!

**Comment by Bonrob2** [comment]
Replaced references to Helm with appropriate new names and conventions for prose and directories, etc.
